features/step_definitions/barclamp_ceph/ceph_steps.rb

```diff
@@ -0,0 +1,108 @@
+Given(/^the barclamp proposal for "([^"]*)" is deployed$/) do |package_name|
```

Comment:
As I mentioned above, replacing `"([^"]*)"` with `ceph` and removing the argument `|package_name|` which is not used anyway we get what is needed here.